cludo/root.go

```diff
@@ -36,8 +36,8 @@ func MakeRootCmd(exit func(int)) (*cobra.Command, error) {
 	viper.BindPFlag("target", rootCmd.PersistentFlags().Lookup("target"))
 	rootCmd.PersistentFlags().Bool("interactive", true, "Should the CLI prompt the user for input")
 	viper.BindPFlag("client.default.interactive", rootCmd.PersistentFlags().Lookup("interactive"))
-	rootCmd.PersistentFlags().StringP("key-path", "k", "~/.ssh/id_rsa", "Path to SSH private key")
-	viper.BindPFlag("client.key_path", rootCmd.PersistentFlags().Lookup("key-path"))
+	rootCmd.PersistentFlags().StringSliceP("ssh-key-paths", "k", []string{"~/.ssh/id_rsa"}, "A comma seperated list of SSH private key filesystem paths")
```

Comment:
Is"~/.ssh/id_rsa" the default or used as an example?  I don't think we want to include a default here.

Reply:
`~/.ssh/id_rsa` is the default for `cludo` and also matches the default key name generate by `ssh-keygen`.

```sh
$ ssh-keygen
Generating public/private rsa key pair.
Enter file in which to save the key (/Users/spkane/.ssh/id_rsa):
```

We can certainly remove it if you'd really prefer. I'd be curious to know what you're concerned about.

Also, it is worth noting that RSA is currently the only key type that is supported by `cludo` at the moment.

Reply:
Forgot that RSA is the only format - thanks for reminding me.

`id_rsa` is the default key for `ssh-keygen`, but having cludo auto-find that feels a little too clever.  I'd rather that we remove any defaults, but give an example `.cludo.yaml` that points at that key.

Reply:
HOWEVER - that's not part of this PR.  merging.